gems/activesupport/6.0/activesupport-generated.rbs

```diff
@@ -10782,81 +10782,6 @@ module ActiveSupport
   end
 end
 
-module ActiveSupport
```

Comment:
I moved this definition to `activesupport.rbs` and modified it. Is this approach okay?